onnxruntime/test/contrib_ops/attention_op_test.cc

```diff
@@ -1865,6 +1873,13 @@ static void RunModelWithRandomInput(
     std::vector<int32_t>& mask_index_data,
     std::string& onnx_model,
     bool is_float16) {
+  // ORT enables TF32 in GEMM for A100. TF32 will cause precsion loss and fail this test.
+  // Do not run this test unless TF32 is disabled explicitly.
+  if (HasCudaEnvironment(800) && ParseEnvironmentVariableWithDefault<int>("NVIDIA_TF32_OVERRIDE", 1) != 0) {
```

Comment:
skip only when is_float16 is false.

Reply:
The float16 model test is flicky right now. I will take a look later and re-enable float16 model test for A100 when that issue is resolved.